src/test/java/com/salesforce/storm/spout/dynamic/DynamicSpoutTest.java

```diff
@@ -565,9 +566,6 @@ public void testResumingSpoutWhileSidelinedVirtualSpoutIsActive() throws Interru
         TopologyContext topologyContext = new MockTopologyContext();
         MockSpoutOutputCollector spoutOutputCollector = new MockSpoutOutputCollector();
 
```

Comment:
This instance was created and opened, but never used as far as I can tell, and never closed.  I opted to prune to remove it since I saw no purpose.

Reply:
Cool, probably an artifact of all our refactoring.